packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.m

```diff
@@ -0,0 +1,86 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import "async_handlers.h"
+#import <Flutter/Flutter.h>
+
+#if !__has_feature(objc_arc)
+#error File requires ARC to be enabled.
+#endif
+
+static NSDictionary *wrapResult(NSDictionary *result, FlutterError *error) {
```

Comment:
Can we add `<String, id>` generics?
`wrapResult` can also be prefixed.

Reply:
done, what do you mean prefixed?

Reply:
However the class prefixing is working (`Api2` in your example if that's the prefix?  I'm not familiar with this test code, so `Api2WrapResult`) could be applied to `wrapResult` since that's such a generic function name that could possibly be shadowing something else.

Reply:
Ahh ok.  The function is declared `static` which means it can't collide with other functions named `wrapResult` in the linker.  If someone added a `wrapResult` function to one of the Flutter headers there could be a collision.  We would never add a function with such a name to a Flutter header.  Any function declared in a header has to meet the higher standards to avoid colliding.